partner_company_type/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,2 @@
+* Denis Roussel <denis.roussel@acsone.eu> (https://acsone.eu)
+* Gilles Meyomesse <gilles.meyomesse@acsone.eu> (https://acsone.eu)
```

Comment:
Please add your name to contributers.

Reply:
It is ok. I almost did nothing to migrate.